content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -132,11 +144,11 @@ Enables anonymous requests to the Kubelet server. Requests that are not rejected
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-Use the TokenReview API to determine authentication for bearer tokens. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)
+Use the TokenReview API to determine authentication for bearer tokens. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's --config flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
 -->
 使用 <code>TokenReview</code> API 对持有者令牌进行身份认证。
-已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
-（<a href="https://kubernetes.io/zh/docs/tasks/administer-cluster/kubelet-config-file/">进一步了解</a>）
+（已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
+请参阅 <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
```

Comment:
ditto

Reply:
Hey, I think it's unnecessary (and annoying) to repeat the same mistake for 20 times. The author is human, they will think/review after accepted one suggestion.

BTW, I have [already suggested](https://github.com/kubernetes/website/pull/34244#pullrequestreview-1003511574) that.

Reply:
thanks to point this, I replace them for existing url